CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/geospatial/AbstractGeoExportPlugin.java

```diff
@@ -461,8 +461,11 @@ public void read(final GraphReadMethods graph, final PluginInteraction interacti
                 throw new PluginException(PluginNotificationLevel.ERROR, "Invalid element type");
         }
 
-        try {
-            exportGeo(parameters, GraphNode.getGraphNode(graph.getId()).getDisplayName(), shapes, attributes, output);
+        try {            
+            //#1646: Check for valid path

```

Comment:
Minor but are you able to omit the ticket number in this comment? Just thinking that it might be a bit confusing if you don't have the context that this number is a ticket number (I think removing will be better than clarifying the context)

Reply:
Done